dashboard/src/components/AppList/AppListItem.tsx

```diff
@@ -58,7 +60,12 @@ function AppListItem(props: IAppListItemProps) {
   return (
     <InfoCard
       key={app.installedPackageRef?.identifier}
-      link={url.app.apps.get(cluster, app.installedPackageRef?.context?.namespace || "", app.name)}
+      link={url.app.apps.get(
+        cluster,
+        app.installedPackageRef?.context?.namespace || "",
+        app.name,
+        pluginObj,
```

Comment:
Similar here in that I wonder if it'd be cleaner for us to use an `InstalledPackageRef` as the arg here?

Also, related to that, isn't it `app.installedPackageRef?.identifier` rather than `app.name` that we should be using here? (for helm they'll be identical, but that's up to the plugin). Hmm, not sure if that applies given this is not an api url, but a frontend hash url. We still want them to be unique, right? (and a different plugin may define the identifier differently from the name to maintain uniqueness).

Reply:
Yes, I missed that one. Thanks!